__docs__/wonder-blocks-link/link.stories.tsx

```diff
@@ -479,6 +479,30 @@ Navigation.parameters = {
     },
 };
 
+export const WithTitle: StoryComponentType = () => (
+    <Body>
+        <Link href="#" title={"I am a title 😎"}>
```

Comment:
**nit:** Since you're just passing a string literal here, you don't need the braces. It doesn't matter if you leave them either. Up to you if you change this:
```suggestion
        <Link href="#" title="I am a title 😎">
```